pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -35,3 +36,27 @@ def my_method1(self, variable1):
 
     def my_method2(self, variable2):
         return variable2
+
+    def my_method6(self):
+        user_id = 1
+        if user_id != 99:
+            # Method without translation
+            raise UserError('String without translation')
+
+    def my_method7(self):
+        user_id = 1
+        if user_id != 99:
+            # Method with translation
+            raise UserError(_('String with translation'))
+
+    def my_method8(self):
+        user_id = 1
+        if user_id != 99:
+            str_error = 'String with translation 2'  # Don't check
```

Comment:
This case is also incorrect, although I know that this can be hard to be detected.